plugin-api.d.ts

```diff
@@ -1177,22 +1168,22 @@ interface BaseFrameMixin
     ExportMixin,
     IndividualStrokesMixin {
   layoutMode: 'NONE' | 'HORIZONTAL' | 'VERTICAL'
-  primaryAxisSizingMode: 'FIXED' | 'AUTO' // applicable only if layoutMode != "NONE"
```

Comment:
See https://www.figma.com/plugin-docs/api/node-properties

All properties have a comment like `Applicable only on auto-layout frames, ignored otherwise.`